### DISCUSSION TITLE
IPTV IGMP issue after 16 days stable

### DISCUSSION THREAD

**Discussion by rdebeuze** [fabianishere/udm-iptv#145] [General]
Hello,

Today after 16 days running stable tv screen freeze frequently and shows error code stb-nmc-400. I have tried to reboot udm pro and the tv boxes but that didn't work. I have checked VLAN interface optained a IP adres and obtained a route from DHCP server.

In the IGMP Proxy log i see s few strange thing but i am familiar enough with this kind of log to analyze but when i see this behaviour on the tv for me it points to IGMP Proxy:

-- Logs begin at Sun 2022-07-17 21:50:31 CEST, end at Sun 2022-07-17 21:56:46 CEST. --
Jul 17 21:50:31 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machine.
Jul 17 21:50:31 ubnt udm-iptv[62]: Obtaining IP address for VLAN interface
Jul 17 21:50:31 ubnt udm-iptv[62]: udhcpc (v1.22.1) started
Jul 17 21:50:31 ubnt udm-iptv[62]: Sending discover...
Jul 17 21:50:31 ubnt udm-iptv[62]: Sending select for 10.134.165.7...
Jul 17 21:50:31 ubnt udm-iptv[62]: Lease of 10.134.165.7 obtained, lease time 3600
Jul 17 21:50:31 ubnt udm-iptv[62]: NATing IPTV network ranges (if necessary)
Jul 17 21:50:31 ubnt udm-iptv[62]: Setting up igmpproxy
Jul 17 21:50:32 ubnt udm-iptv[62]: Starting igmpproxy
Jul 17 21:50:32 ubnt igmpproxy[62]: can't join group 239.255.3.22 on interface iptv; Errno(19): No such device
Jul 17 21:50:33 ubnt igmpproxy[62]: can't join group 233.89.188.1 on interface iptv; Errno(19): No such device
Jul 17 21:50:33 ubnt igmpproxy[62]: No interfaces found for source 192.168.255.249
Jul 17 21:50:33 ubnt igmpproxy[62]: No interfaces found for source 192.168.255.249
Jul 17 21:50:35 ubnt igmpproxy[62]: can't join group 239.254.127.63 on interface iptv; Errno(19): No such device
Jul 17 21:50:35 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:50:35 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:51:57 ubnt igmpproxy[62]: No interfaces found for source 0.0.0.0
Jul 17 21:56:43 ubnt systemd[1]: Stopping IPTV support for the UniFi Dream Machine...
Jul 17 21:56:43 ubnt systemd[1]: udm-iptv.service: Succeeded.
Jul 17 21:56:43 ubnt systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Jul 17 21:56:43 ubnt systemd[1]: udm-iptv.service: Consumed 276ms CPU time.
Jul 17 21:56:43 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machine.
Jul 17 21:56:43 ubnt udm-iptv[12870]: Device iptv already exists.. Deleting device
Jul 17 21:56:43 ubnt udm-iptv[12870]: Obtaining IP address for VLAN interface
Jul 17 21:56:43 ubnt udm-iptv[12870]: udhcpc (v1.22.1) started
Jul 17 21:56:43 ubnt udm-iptv[12870]: Sending discover...
Jul 17 21:56:43 ubnt udm-iptv[12870]: Sending select for 10.134.165.18...
Jul 17 21:56:43 ubnt udm-iptv[12870]: Lease of 10.134.165.18 obtained, lease time 3600
Jul 17 21:56:43 ubnt udm-iptv[12870]: NATing IPTV network ranges (if necessary)
Jul 17 21:56:43 ubnt udm-iptv[12870]: Setting up igmpproxy
Jul 17 21:56:43 ubnt udm-iptv[12870]: Starting igmpproxy
Jul 17 21:56:43 ubnt udm-iptv[12870]: adding VIF, Ix 0 Fl 0x0 IP 0x01b4a8c0 br0, Threshold: 1, Ratelimit: 0
Jul 17 21:56:43 ubnt udm-iptv[12870]: adding VIF, Ix 1 Fl 0x0 IP 0x12a5860a iptv, Threshold: 1, Ratelimit: 0
Jul 17 21:56:43 ubnt udm-iptv[12870]: Joining group 224.0.0.2 on interface br0
Jul 17 21:56:43 ubnt udm-iptv[12870]: Joining group 224.0.0.22 on interface br0
Jul 17 21:56:43 ubnt udm-iptv[12870]: RECV Membership query   from 192.168.180.1   to 224.0.0.1
Jul 17 21:56:43 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.1   to 224.0.0.2
Jul 17 21:56:43 ubnt udm-iptv[12870]: The IGMP message was from myself. Ignoring.
Jul 17 21:56:43 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.1   to 224.0.0.22
Jul 17 21:56:43 ubnt udm-iptv[12870]: The IGMP message was from myself. Ignoring.
Jul 17 21:56:43 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.105 to 233.89.188.1
Jul 17 21:56:43 ubnt udm-iptv[12870]: Inserted route table entry for 233.89.188.1 on VIF #0
Jul 17 21:56:43 ubnt udm-iptv[12870]: Joining group 233.89.188.1 on interface iptv
Jul 17 21:56:43 ubnt udm-iptv[12870]: The source address 192.168.180.105 for group 233.89.188.1 is from downstream VIF[0]. Ignoring.
Jul 17 21:56:44 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.191 to 239.254.127.63
Jul 17 21:56:44 ubnt udm-iptv[12870]: Inserted route table entry for 239.254.127.63 on VIF #0
Jul 17 21:56:44 ubnt udm-iptv[12870]: Joining group 239.254.127.63 on interface iptv
Jul 17 21:56:45 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.60  to 239.255.255.251
Jul 17 21:56:45 ubnt udm-iptv[12870]: Inserted route table entry for 239.255.255.251 on VIF #0
Jul 17 21:56:45 ubnt udm-iptv[12870]: Joining group 239.255.255.251 on interface iptv
Jul 17 21:56:46 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.13  to 239.0.0.250
Jul 17 21:56:46 ubnt udm-iptv[12870]: Inserted route table entry for 239.0.0.250 on VIF #0
Jul 17 21:56:46 ubnt udm-iptv[12870]: Joining group 239.0.0.250 on interface iptv

Also create a service log:

=== Service Logs ===
Jul 17 22:05:52 ubnt udm-iptv[12870]: Updated route entry for 233.89.188.1 on VIF #0
Jul 17 22:05:52 ubnt udm-iptv[12870]: The source address 192.168.180.105 for group 233.89.188.1 is from downstream VIF[0]. Ignoring.
Jul 17 22:05:52 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.13  to 239.0.0.250
Jul 17 22:05:52 ubnt udm-iptv[12870]: Updated route entry for 239.0.0.250 on VIF #0
Jul 17 22:05:54 ubnt udm-iptv[12870]: The IGMP message was local multicast. Ignoring.
Jul 17 22:05:54 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.60  to 239.255.255.251
Jul 17 22:05:54 ubnt udm-iptv[12870]: Updated route entry for 239.255.255.251 on VIF #0
Jul 17 22:05:55 ubnt udm-iptv[12870]: RECV V2 member report   from 192.168.180.254 to 239.254.127.63
Jul 17 22:05:55 ubnt udm-iptv[12870]: Updated route entry for 239.254.127.63 on VIF #0
Jul 17 22:05:55 ubnt udm-iptv[12870]: The IGMP message was local multicast. Ignoring.

Is there somting i can check of change to make this work again.

**Comment by rdebeuze** [comment]
Found it! In log a saw interface br0 but in my case this should be br185 strange that is changed today without any changes to config in 16 day but i works again!

**Reply by fabianishere** [reply]
It could be a firmware upgrade in the background that reset the configuration to default.

**Reply by rdebeuze** [reply]
Ok thankyou will keep that in mind

**Reply by RonaldBlom** [reply]
how did you find out which br interface to use?

**Reply by RonaldBlom** [reply]
that is the vlan id used to place your tvboxes in?

**Reply by rdebeuze** [reply]
Yes the br interface number is your VLAN ID. In my case i made a network 192.168.185.0/24 with VLAN id 185 so my vlan interface is br185
